tsconfig.json

```diff
@@ -3,12 +3,13 @@
     "declaration": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
+    "lib": ["DOM", "ES2020"],
     "module": "CommonJS",
     "moduleResolution": "node",
     "useUnknownInCatchVariables": false,
     "sourceMap": true,
     "strict": true,
-    "target": "ES2017"
+    "target": "ES2020"
```

Comment:
Can you leave a comment in the PR description that this is changed and why?

Reply:
This is purely to align with `snaps-monorepo` and unlock ES2020 functionalities

Reply:
Ok, thanks!